src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/ProjectConfigurationFactory.cs

```diff
@@ -28,9 +29,9 @@ public static ProjectConfiguration Create(string configuration)
             return Create(configuration, dimensionsBuilder.ToImmutable());
         }
 
-        public static IImmutableSet<ProjectConfiguration> CreateMany(params string[] configurations)
+        public static IReadOnlyList<ProjectConfiguration> CreateMany(params string[] configurations)
         {
-            var configurationsBuilder = ImmutableHashSet.CreateBuilder<ProjectConfiguration>();
+            var configurationsBuilder = ImmutableArray.CreateBuilder<ProjectConfiguration>();
```

Comment:
This change is not for public API. This is just for the Mock. So this change is ok and I guess, we dont need to do anything.